opensrp-chw/src/kk/resources/anc_member_registration.properties

```diff
@@ -83,13 +83,13 @@ anc_member_registration.step1.health_insurance.hint = Does the pregnant mother h
 anc_member_registration.step1.health_insurance.values[0] = Yes
 anc_member_registration.step1.health_insurance.values[1] = No
 
-anc_member_registration.step1.accessibility_issues.hint = Does the woman experience difficulty in doing any of the following? 
-anc_member_registration.step1.accessibility_issues.values[0] = Seeing
-anc_member_registration.step1.accessibility_issues.values[1] = Hearing
-anc_member_registration.step1.accessibility_issues.values[2] = Walking or climbing stairs
-anc_member_registration.step1.accessibility_issues.values[3] = Remembering or concentrating
-anc_member_registration.step1.accessibility_issues.values[4] = Communicating with others
-anc_member_registration.step1.accessibility_issues.values[5] = Other
+anc_member_registration.step1.accessibility_issues.label = Does the woman experience difficulty in doing any of the following? 
+anc_member_registration.step1.accessibility_issues.options.chk_seeing.text = Seeing
+anc_member_registration.step1.accessibility_issues.options.chk_hearing.text  = Hearing
+anc_member_registration.step1.accessibility_issues.options.chk_walking_or_climbing_stairs.text = Walking or climbing stairs
+anc_member_registration.step1.accessibility_issues.options.chk_remembering_or_concentrating.text = Remembering or concentrating
+anc_member_registration.step1.accessibility_issues.options.chk_communicating_with_others.text = Communicating with others
+anc_member_registration.step1.accessibility_issues.options.chk_none.text = None
```

Comment:
I think the last one is Other and not none and also we should have a relevance to give the user an option to enter the value incase they select the "Other" option